### ISSUE TITLE
cannot pass an existing pool to PostgresDialect

### ISSUE THREAD

**Issue by DanielFGray** [koskimas/kysely#81]
for example in my app I use `express-session` and `connect-pg-simple`, the latter requires passing a pg pool
 
 ideally I would like to re-use that pool when creating a kysely instance rather than creating a new pool

**Comment by koskimas** [comment]
This issue is formatted like bug 😄 Why do you expect this to exist? I can't read your mind you know 😂 

As a __new feature__ this is a good idea though. I'll make this possible soon.

**Comment by koskimas** [comment]
Once the next version is released, you can do this:

```ts
import { Pool } from 'pg'

new Kysely<DB>({
  dialect: new PostgresDialect(
    new Pool(config)
  )
})
```

**Comment by koskimas** [comment]
Released in 0.18.1
